### PR TITLE
Page: Remove bottom page margin 

### PR DESCRIPTION
**Pull request by torkelo**
Removes the bottom page margin: 
Screenshot from 2023-04-04 09-18-39

### CODE REVIEW

**Review by ashharrison90 — COMMENTED** [review]
sorry! completely missed this somehow... these changes are fine from a code point of view, but we definitely need a UX pov on this too.

my biased opinion: the previous style is more visually appealing. to the point where i'm not sure it's worth the trade off to gain an extra 20-30px.

**Comment by torkelo** [comment]
@ashharrison90 yea, I am also torn. But for small laptops every pixel helps (esp horizontally)

**Comment by ashharrison90** [comment]
yep it's tricky. tried playing around with reducing margins/padding a bit, and whatever you do ends up looking a little cramped 😬

**Comment by amy-super** [comment]
I think the update helps with critiques of Grafana that say that things are too "boxy" - but that said, it doesn't personally bother me to have the margin. If we want to optimize our vertical space, this helps a bit but we should be aware that it's not going to be a gamechanger. Does this impact anything in Explore or on a dashboard? I feel like those are the places where space is the most precious. 

so tl;dr - as long as it doesn't break things, I'm fine with these changes but personally don't have really strong opinions about them.

**Comment by torkelo** [comment]
@amy-super it does not impact explore as explore does not use this page layout (Explore uses the "canvas" layout, where pages does not have a section nav and can place content directly on the canvas) 

Another solution that I kind of like more is to just remove the right side margin and the bottom margin (They are the ones that bother me the most as they don't add much and just take up space). 

Screenshot 2023-04-03 at 16 24 23

**Review by ashharrison90 — APPROVED** [review]
looks good to me 🥳
